corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media.html

```diff
@@ -1,8 +1,8 @@
 {% load i18n %}
 {% load hq_shared_tags %}
-<div id="media_image">
+<div id="{{ qualifier|default_if_none:"" }}media_image">
     <div class="control-group">
-        <label class="control-label" for="menu_image_path">{% trans "Icon" %}</label>
+        <label class="control-label" for="menu_image_path">{% trans ICON_LABEL %}</label>
```

Comment:
I tried a few ways to get default values (including what you suggest) but without success.

The all caps following what had been done elsewhere e.g. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/partials/case_list_setting.html#L8-8